modules/quotas-requiring-explicit-quota.adoc

```diff
@@ -8,44 +8,100 @@
 = Requiring explicit quota to consume a resource
 
 If a resource is not managed by quota, a user has no restriction on the amount
-of resource that can be consumed.  For example, if there is no quota on storage
+of resource that can be consumed. For example, if there is no quota on storage
 related to the gold storage class, the amount of gold storage a project can
 create is unbounded.
 
-For high-cost compute or storage resources, administrators might want to require
-an explicit quota be granted in order to consume a resource.  For example, if a
-project was not explicitly given quota for storage related to the gold storage
-class, users of that project would not be able to create any storage of that
-type.
+In {product-title} {product-version} it is not possible to configure the `ResourceQuota` admission plug-in. Instead, explicit resource quotas can be defined for a specific resource in the default project template. Those resource quotas are then applied to newly created projects.
+
+[NOTE]
```

Comment:
I have now removed this and replaced it with the single sentence suggested in your earlier comment.

Reply:
@kalexand-rh reading this comment again, I am unsure if this is in reference to the note or the paragraph preceding it. Can you please confirm?

Reply:
On reflection I think this comment relates to the note, given the position of the comment. I included that note because the Bugzilla mentions that this functionality is possible through webhook admission plug-ins. On review, I agree it is confusing. Plus, it doesn't really add anything without a procedure showing the reader how to do this. That procedure would be very detailed and technical. As such, I will remove the note so that the document can focus on providing the simpler method.